src/Http/Middleware/DetermineAndSetStockId.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Rapidez\Msi\Http\Middleware;
+
+use Closure;
+use Illuminate\Support\Facades\DB;
+
+class DetermineAndSetStockId
+{
+    /**
+     * @param $request
+     * @param Closure $next
+     * @return mixed
+     */
+    public function handle($request, Closure $next)
+    {
+        $stockId = DB::table('inventory_stock_sales_channel')
+            ->where('inventory_stock_sales_channel.type', 'website')
```

Comment:
Do we know this will always be website? If so this will be fine, if not perhaps we could use the mage_run_type
https://devdocs.magento.com/guides/v2.4/config-guide/multi-site/ms_over.html#:~:text=variables%20as%20follows%3A-,MAGE_RUN_TYPE,-can%20be%20either

If i remember correctly this is store by default

Reply:
@indykoning according to Magento it should be possible to match a stocks with websit, customer, groups, etc. to form a sales channel. But I can't seem to find the proper documentation for that. I currently can set website and it's also the default type. https://github.com/magento/inventory/blob/develop/InventorySalesApi/Api/Data/SalesChannelInterface.php#L29

Reply:
@indykoning according to Magento it should be possible to match a stocks with website, customer, groups, etc. to form a sales channel. But I can't seem to find the proper documentation for that. I currently can set website and it's also the default type. https://github.com/magento/inventory/blob/develop/InventorySalesApi/Api/Data/SalesChannelInterface.php#L29